### DISCUSSION TITLE
Walls can not be taught to organisms with brains, only evolved

### DISCUSSION THREAD

**Discussion by SelfSealShelf** [MaxRobinsonTheGreat/EvolutionSimulatorV2#61] [Ideas and Suggestions]
[comment]: <> (ISSUES ARE NOT FOR IDEAS OR SUGGESTIONS)

**Describe the bug**
Organisms can evolve to see and recognize walls, but cannot be taught them manually in the editor, making reproducing certain behaviors difficult or impossible

**To Reproduce**
Line an environment with walls and promote mover development until they have eyes and can recognize the boundaries of their environment.

**Expected behavior**
Walls should be selectable when constructing an organism

**Comment by MaxRobinsonTheGreat** [comment]
I consider this a feature request, not a bug
